app/api/v1/questions/questions.py

```diff
@@ -0,0 +1,26 @@
+import flask
+from flask import request,jsonify,abort,make_response
+
+app = flask.Flask('__name__')
+app.config['DEBUG'] = True
+
+#add dummy questions
+questions = [{'id':1,
```

Comment:
I would you like for you to consider placing the questions list into its equivalent question model class. Like this:
```python
QUESTIONS = []

class QuestionModel:
    def __init__(self, title, description):
        self.id = len(QUESTION)  + 1
        self.title = title
        self.description = description

    '''
    Add other methods to access the QUESTIONS list 
    like get_questions(), get_question_by_id(), etc.
    Depending on your implementation details, they methods may be
    class methods, instance methods or static methods.
    '''
```
**NOTE**:  _You need also to provide a method for converting the QuestionModel class to a dictionary._

Reply:
Thanks Khwilo for your feedback . i havent started on implimentation but i will take that point into consideration when i start. i was only checking if the endpoints are working